ystia/yorc
# Emit an event when purge is actually done.

# Pull Request description


## Description of the change

Emit an event when purge is actually done.
Remove logs & events before deploying a new
application to remove those stale purged events

### How to verify it

Should use the corresponding plugin PR ystia/yorc-a4c-plugin#123.

Deploy an application and undeploy it using the CLI with the purge option.
Then redeploy the same application...

### Description for the changelog

* Emit a persistent event on deployment purge ([GH-402](https://github.com/ystia/yorc/issues/402))

## Applicable Issues

Fixes #402
